base/linalg/linalg.jl

```diff
@@ -267,7 +278,7 @@ The reason for this is that factorization itself is both expensive and typically
 and performance-critical situations requiring `rdiv!` usually also require fine-grained
 control over the factorization of `B`.
 """
-rdiv!(Y, A, B)
+rdiv!(A, B)
```

Comment:
Does a three-argument `rdiv!` docstring still exist somewhere?

Reply:
There is no 3-arg `rdiv!`

Reply:
Now that's amusing 😄.